Rakefile

```diff
@@ -345,6 +345,15 @@ begin
   # Task Release
   #-----------------------------------------------------------------------------#
 
+  desc "Add owners on RubyGems"
+  task :add_owners, :gem_name do |t, args|
+    gem_name = args[:gem_name]
+    owners = %w(segiddins@segiddins.me boris@buegling.com gems@mariusrackwitz.de)
```

Comment:
I think these should become an argument instead of hard-coded into the script?

Reply:
> I think these should become an argument instead of hard-coded into the script?